src/registrar/templates/application_about_your_organization.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'application_form.html' %}
+{% load field_helpers %}
+
+{% block form_instructions %}
+  <p>[For special districts, interstate governments]</p>
```

Comment:
Would this be better as an h2 (like the other pages), without the square brackets? Ignore this though if this matches the design requirements

Reply:
That line should be removed altogether. It was a content design note and shouldn't appear as text in the UI.